kolibri/core/tasks/registry.py

```diff
@@ -347,6 +358,7 @@ def enqueue_at(
             interval=interval,
             repeat=repeat,
             retry_interval=retry_interval,
+            ignore_running=ignore_running,
```

Comment:
Rather than plumbing an `ignore_running` argument through every wrapper and call site to control whether the exception is raised, would it be cleaner to catch and ignore the exception on the few call sites where you’re setting `ignore_running=True`?

Reply:
I'd rather use the try..except because the case raising the exception is quite clear and clean. I would not add one more param to the api (involving adding the same line of code in several places) for a situation that can be easily determined without needing it. 
The parameter makes the api call more readable but forces adding code that's not making the code more readable than the exception.